provisioning/security/tpm-provider/src/main/java/com/microsoft/azure/sdk/iot/provisioning/security/hsm/SecurityProviderTPMHsm.java

```diff
@@ -393,10 +392,9 @@ else if (rc != TPM_RC.HANDLE)
     /**
      * Getter for extracting StorageRootKey from TPM
      * @return The StorageRootKey from TPM
-     * @throws SecurityProviderException If StorageRootKey could not be extracted.
      */
     @Override
-    public byte[] getStorageRootKey() throws SecurityProviderException
```

Comment:
Same comment for these two methods. They are typically only used internally, but this class is directly used by customers, so it would risk breaking them